app/src/main/res/layout/user_profile_fragment.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".ui.user.UserFragment">
+
+    <!-- TODO: Update blank fragment layout -->
+    <TextView
```

Comment:
Has this to be empty? Can't this be removed? 🤔

Reply:
Looks also like a template thing